css/main.css

```diff
@@ -416,41 +388,98 @@ a:hover {
   .general,
   .research {
     width: 85%;
-    margin: 0px 3px; }
-  .track-text {
-    padding: 0px 20px; } }
+    margin: 0px 3px; } }
 
 @media (max-width: 500px) {
   .tracks {
     margin-bottom: 0; }
   .gallery {
-    height: 540px; }
-  .track-text {
-    padding: 0px 20px;
-    font-size: 95%; } }
+    height: 540px; } }
 
 @media (max-width: 425px) {
   .tracks {
     margin-bottom: 0; }
   .gallery {
-    height: 580px; }
-  .track-text {
-    padding: 0px 20px;
-    font-size: 95%; } }
+    height: 580px; } }
 
 @media (max-width: 385px) {
   .gallery {
-    height: 640px; }
-  .track-text {
-    padding: 0px 20px;
-    font-size: 95%; } }
+    height: 640px; } }
 
 @media (max-width: 280px) {
   .gallery {
-    height: 640px; }
-  .track-text {
-    padding: 0px 23px;
-    font-size: 85%; } }
+    height: 640px; } }
+
+.tracks-card {
+  font-size: 1rem;
+  padding: 1.3rem;
+  border-radius: 2rem;
+  box-shadow: 7px 7px 10px #DCDCDC;
```

Comment:
Yep right here - I would get rid of the shadow. Though if you HAVE to keep it, I'd inch it a little bit closer to the box. Otherwise it gives me 2010 website vibes hehe. Other than that I don't see any issues. Good work!!

Reply:
Haha I will remove it!! Thanks for the feedback :)